### DISCUSSION TITLE
How to force https protocol in dev mode?

### DISCUSSION THREAD

**Discussion by laszlo1337** [sveltejs/kit#10962] [Q&A] [answered]
I am trying to implement Instagram oauth2 integration but they only allow https redirects, so I cannot test it locally and remote server doesn't exist yet. 

Is it possible to configure SvelteKit dev mode to launch using https? I couldn't find anything in the docs or issues or in here.

Thanks.

**Comment by david-plugge** [comment, marked as answer]
Take a look at this
https://vitejs.dev/config/server-options.html#server-https
